internal/cli/atlas/clusters/watch.go

```diff
@@ -44,9 +45,14 @@ func (opts *WatchOpts) initStore(ctx context.Context) func() error {
 
 func (opts *WatchOpts) watcher() (bool, error) {
 	result, err := opts.store.AtlasCluster(opts.ConfigProjectID(), opts.name)
+	if opts.previousStatus == "UPDATING" && err != nil {
```

Comment:
how about storing a flag `allowExponentialBackoff`? this way we won't need the custom error or previous status, correct?

Reply:
Updated